### ISSUE TITLE
[2.3-beta-1] "This page isn't responding"

### ISSUE THREAD

**Issue by alangecker** [bigbluebutton/bigbluebutton#11878]
I got now several reports from people, that the BBB sometime gets unresponsive with a browser message
"This page isn't responding". So probably an endless loop somewhere.

Happens with Firefox and Chrome.

I couldn't reproduce it, but it appears to be frequent issue, but wasn't reported here yet.  

So far this issue doesn't help, but did anyone experience the same and is maybe even able to debug it in that moment?

### Steps for debugging
1. Have the site somehow unresponsive
2. Open Developer Tools -> Sources
3. Press Pause-Button
4. Post a screenshot here

**Comment by alangecker** [comment]
I managed to reproduce and debug it a bit!

somehow 

It stucked in the `timeWindowsids.forEach()` loop here, because there are really a lot of keys for `preJoinMessages` (21.384 in the moment of debugging)
https://github.com/bigbluebutton/bigbluebutton/blob/f1fe98b4030b462447c2ec35b657dde2958f01c9/bigbluebutton-html5/imports/ui/components/components-data/chat-context/context.jsx#L210-L230

**Comment by ffdixon** [comment]
Thanks Chandi -- we'll take a closer look.

**Comment by alangecker** [comment]
@ffdixon 
Good point! Since the server is now empty anyway I was able to switch to beta-2 and could not reproduce the error. Might be fixed in beta-2, but I could not find any related change in the release changelog? 🤔

**Comment by Tainan404** [comment]
Hi @alangecker, I made some changes in how the unread works, had a condition generating empty calls for unread messages, This may have gone unnoticed 😄
